### DISCUSSION TITLE
How to view a file as if it was compiled with `target_feature=+avx2,...`?

### DISCUSSION THREAD

**Discussion by MadProbe** [rust-lang/rust-analyzer#16345] [Q&A] [answered]
I am doing development for code part of which is specifically optimized with avx2 under `#[cfg(target_feature = "avx2")]` and I want to see my file with the code as if I have compiled my project with `-C target-feature=+avx2`. How to do it? I am developing in VSCode BTW, if that may help to pin point exact actions for resolving this problem.

**Comment by Veykril** [comment, marked as answer]
I don't think that's currently possible unfortunately.
